### PR TITLE
implement kick_ghosts.py

### PR DESCRIPTION
**Pull request by Anthony271828**
This module loads the Gostkick from the database and manages it.
If no configuration exists, a default configuration is created.

### CODE REVIEW

File: core/error/error_reply.py [hunk 1]
```diff
@@ -19,3 +19,22 @@ async def error_reply(ctx: Context,
...
+    embed.add_field(name="Cause", value=cause, inline=False)
+    embed.add_field(name="Solution", value=solution, inline=False)
+
+    guild: Guild = bot.guilds[0]
+    text_channels: list[TextChannel] = guild.text_channels
+
+    for channel in text_channels:
+        permissions: Permissions = channel.permissions_for(guild.get_member(bot.user.id))
+        if permissions.read_messages and permissions.send_messages:
+            await channel.send(embed=embed)
```
Comment (maxwai) [inline core/error/error_reply.py:39]: Why send it in every possible channel?

File: cogs/kick_ghosts.py [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+            try:
+                for c in self.config:
+                    key = c.value
+                    default = self.config[c]
+
+                    enabled = await self.db.find_one({key: {"$exists": True}})
+                    if enabled:
+                        self.config[c] = enabled[key]
+                    else:
+                        await self.db.insert_one({key: default})
```
Comment (maxwai) [inline cogs/kick_ghosts.py:54]: You don't need the variable `default` since it is only used there

File: cogs/kick_ghosts.py [hunk 3]
```diff
@@ -0,0 +1,291 @@
...
+
+        for member in kick_member:
+            try:
+                await member.kick(reason="Too long without verification")
+            except Forbidden:
+                logger.error(f'Tried to kick User="{member.name}#{member.discriminator}({member.id})" '
+                             f'but failed due missing permissions.')
+                embed = Embed(title="Kick Ghosts",
+                              description=f"Tried to kick <@{member.id}> but failed due missing permissions.")
+                await debug_chat.send(embed=embed)
```
Comment (maxwai) [inline cogs/kick_ghosts.py:216]: Also write message in debug chat if kick was successful

**Review by maxwai — CHANGES_REQUESTED** [review]
Checked from tab s7+ in France at the sea

File: core/error/error_reply.py (hunk 1 above)
Comment (Anthony271828) [reply core/error/error_reply.py:39]: As you can see in line 40 (https://github.com/Anthony271828/HM-DiscordBot/pull/32/files#diff-f546f1102df14e5a98a484b02643aeb68053bb78a15a73b1d6b01a4f425c80c8R40)
Means that the first usable chat is used, not all!

File: cogs/kick_ghosts.py (hunk 2 above)
Comment (Anthony271828) [reply cogs/kick_ghosts.py:59]: Agree.

File: cogs/kick_ghosts.py (hunk 3 above)
Comment (Anthony271828) [reply cogs/kick_ghosts.py]: Adjustment made.

File: core/error/error_reply.py (hunk 1 above)
Comment (maxwai) [reply core/error/error_reply.py:39]: oh yes sorry didn't see

**Review by maxwai — APPROVED** [review]

**Pull request merged**